docs/security/trust/deploying_notary.md

```diff
@@ -21,7 +21,7 @@ The easiest way to deploy Notary Server is by using Docker Compose. To follow th
         docker-compose up -d
 
 
-    For more detailed documentation about how to deploy Notary Server see https://github.com/docker/notary.
+    For more detailed documentation about how to deploy Notary Server see the [instructions to run a Notary service](https://docs.docker.com/notary/running_a_service/) as well as https://github.com/docker/notary for more information.
```

Comment:
this should link to the markdown file - `/notary/running_a_service.md`, so that versioned linking works.